test/conformance/OWNERS

```diff
@@ -0,0 +1,5 @@
+# This is the owner of the test code.  The test data itself is owned by sig-architecture.
+reviewers:
+  - mml
+approvers:
+  - mml
```

Comment:
Seems like we should maintain at least two names here

Reply:
Is that a volunteer I see?  So it is, good sir!